tests/pgautofailover_utils.py

```diff
@@ -194,7 +194,7 @@ def create(self):
         create_command = [shutil.which('pg_autoctl'), 'create',
                           self.role.command(),
                         '--pgdata', self.datadir,
-                        '--nodename', str(self.vnode.address),
+                        #'--nodename', str(self.vnode.address),
```

Comment:
dead code

Reply:
Removed, added a comment instead that we explicitly choose to not pass --nodename.